msteams-platform/tabs/how-to/authentication/tab-sso-overview.md

```diff
@@ -0,0 +1,187 @@
+
+Tabs are Teams-aware web pages. To enable SSO in a web-page hosted inside a tab app, add [Teams Javascript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), and call `microsoftTeams.initialize()`. After you've initialized Microsoft Teams, call `microsoftTeams.getAuthToken()` to get the access token for your app.
+
+### Use cases for enabling SSO
+
+You can enable SSO in Teams for all apps that support Azure AD as an identity provider. In addition to using SSO for authenticating app users in a tab app, you can also use it to enable seamless access across Teams. Some scenarios where you can use `getAuthToken()` to use Teams ID token for allowing access to your app users:
+
+- If your app already exists and you want it to be available in Teams within a tab app, you can use SSO to allow your users access to your app from within the tab app. Based on the app user's identity within Teams, you can obtain access token for them using `getAuthToken()`.
+
+- If your app has configurable tabs, you can use `getAuthToken()` to allow app users into the tab app without signing in again. SSO is applicable to some settings that need to be configured at a configuration stage.
+
+- Your app may have a bot, a tab, a message extension, or Adaptive Cards, and allows your app users to invoke task module from within a bot or a tab. Enabling SSO and using `getAuthToken()` will let your app users access a task module from within a bot app without having to sign in again. Teams can use the app user's Teams identity and obtain an access token from Azure AD.
+
+- You can also use SSO to allow your app users to obtain access to Stage view without need to be authenticated again. The access token obtained for the app user when they first used your app can be used to allow them to use Stage view.
+
+> [!TIP]
+> You can also use the SSO API to authenticate app users in [task modules](../../../task-modules-and-cards/what-are-task-modules.md) that embed web content.
+
+To achieve SSO at runtime, configure your app to enable SSO for authenticating and authorizing app users.
+
+## Enable SSO for a tab app
+
+This section describes the tasks involved in implementing SSO for a tab app. These tasks are language- and framework-agnostic.
+
+To enable SSO for a tab app:
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; :::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/enable-sso.png" alt-text="Steps to enable SSO for tab" border="false" lightbox="../../../assets/images/authentication/teams-sso-tabs/enable-sso.png":::
+
+1. **Register with Azure AD**: Create an Azure AD app to generate an app ID and application ID URI. Configure redirect URI where Azure AD would send the access token in exchange for identity token for the current app user logged into Teams. For generating access token, you configure scopes and OBO flow.
```

Comment:
NEW COMMENT
Neither a redirect URI or OBO Flow are needed in this scenario. The "OBO Flow" is only used in the case of calling Microsoft Graph or some other downstream application; even in this case there is no "OBO Flow" setting in the app registration however an App Secret is needed for this.
The "Redirect URI" is only needed if an application implements OBO Flow and wants to handle consent for the downstream API, such as the app handling consent for User.Read Graph permissions. This is accomplished by using a pop-up (authentication.authenticate() call) that redirects the user to Azure AD where they can sign in and grant consent; then Azure AD redirects back to the app in the popup; the page that will handle this must be registered as the "Redirect URI"